scripts/modeling_toolbox/utils.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+import tqdm
+import numpy as np
+
+def load_data(data_uri, nrows=None):
+    """
+    Function to retrieve data from a given file or URL
+    in a Pandas DataFrame suitable for model training.
+    nrows limits the amount of data
+    """
+    data_df = pd.read_csv(data_uri, nrows=nrows)
+    data_df.index = data_df.id
+    lowercase = lambda x: str(x).lower()
+    data_df.rename(lowercase, axis='columns', inplace=True)
+    data_df.drop(['dimension_x'], axis=1, inplace=True)
+    data_df.rename(columns={'attack':'rendition',
+                            'title':'source',
+                            'dimension': 'dimension_x'},
+                   inplace=True)
+    data_df['rendition'] = data_df['rendition'].apply(lambda x: set_rendition_name(x))
+    data_df['dimension_y'] = data_df['rendition'].apply(lambda x: int(x.replace('p','').split('_')[0]))
+    data_df['size_dimension_ratio'] = data_df['size'] / (data_df['dimension_y'] * data_df['dimension_x'])
```

Comment:
I see elsewhere that `size_dimension_ratio` is calculated as `size / dimension`. Why do we calculate `size_dimension_ratio` as `size / dimension_y * dimension_x` here?

Reply:
I think it's to better maintain feature scale for 2D features. Models are trained with features scaled like that now. Maybe @ndujar can comment.